tools/build_test.py

```diff
@@ -0,0 +1,100 @@
+import sys
+from os.path import join, abspath, dirname, exists, basename
+ROOT = abspath(join(dirname(__file__), ".."))
+sys.path.insert(0, ROOT)
+
+import argparse
+
+from export import EXPORTERS
```

Comment:
We can make another PR that fixes imports to be PEP8 style across the codebase. For now, let's not make that soon-to-be-PR any bigger than it has to be.